pkg/cmd/tknpac/generate/templates/go.yaml

```diff
@@ -2,7 +2,7 @@
 apiVersion: tekton.dev/v1beta1
 kind: PipelineRun
 metadata:
-  name: go-template
```

Comment:
this will break operator logic for creating configmap

Reply:
lets merge this once you are done ...
I will update in operator.

Reply:
okay, yeah the name looks better imho

Reply:
(it's not a template and the user doesnt care about it)